Libplanet.Tools/Mpt.cs

```diff
@@ -166,6 +167,68 @@ public class Mpt
             configurationService.Store(configuration);
         }
 
+        [Command(Description="List all aliases stored.")]
+        public void List(
+            [FromService] IConfigurationService<ToolConfiguration> configurationService)
+        {
+            ToolConfiguration configuration = configurationService.Load();
+            Dictionary<string, string> aliases = configuration.Mpt.Aliases;
+
+            int maxAliasLength = aliases.Keys.Max(alias => alias.Length),
+                maxPathLength = aliases.Values.Max(alias => alias.Length);
+            Console.Error.WriteLine(
+                $"{{0,-{maxAliasLength}}} | {{1,-{maxPathLength}}}",
+                "ALIAS",
+                "PATH");
+            Console.Error.WriteLine(new string('-', 3 + maxAliasLength + maxPathLength));
+            foreach (KeyValuePair<string, string> pair in aliases)
+            {
+                Console.Write($"{{0,-{maxAliasLength}}} ", pair.Key);
+                Console.Error.Write("| ");
+                Console.WriteLine($"{{0,-{maxPathLength}}}", pair.Value);
+            }
```

Comment:
Without flushing, the order of chunks between two different streams could be mixed.

```suggestion
            Console.Error.WriteLine(new string('-', 3 + maxAliasLength + maxPathLength));
            Console.Error.Flush();
            foreach (KeyValuePair<string, string> pair in aliases)
            {
                Console.Write($"{{0,-{maxAliasLength}}} ", pair.Key);
                Console.Out.Flush();
                Console.Error.Write("| ");
                Console.Error.Flush();
                Console.WriteLine($"{{0,-{maxPathLength}}}", pair.Value);
                Console.Out.Flush();
            }
```

Reply:
I amended it in b7471f2.